src/main.rs

```diff
@@ -351,17 +364,14 @@ fn truncated_description(description: &str) -> String {
         .split(".")
         .filter(|sentence| !sentence.trim().is_empty());
 
-    let mut out = format!(
-        "{}.",
-        lossy_unicode(&sentences.next().unwrap_or_default().as_bytes())
-    );
+    let out = sentences.next().unwrap_or_default().trim_end_matches('.');
```

Comment:
Why remove the trailing `.`? Was this for a particular test case? Some of the changes in the function cause the first assertion of `test_truncated_description` to fail.

Reply:
Oh, I took it from upstream python but I forgot to update the tests. I was wondering why upstream python does not have the trailing `.` and noticed that ours is probably a bit old. I fixed the behavior to only remove the last period rather than the first one.